common/display/virtualdisplay.cpp

```diff
@@ -198,6 +198,13 @@ bool VirtualDisplay::Present(std::vector<HwcLayer *> &source_layers,
 
       const HwcRect<int> &display_frame = layer->GetDisplayFrame();
       sf_handle = layer->GetNativeHandle();
+
+      if (NULL == sf_handle) {
+        ETRACE("layer->GetNativeHandle returns NULL, skip layer_index: %u.",
+               layer_index);
+        continue;
+      }
+
       std::shared_ptr<OverlayBuffer> buffer(NULL);
       uint32_t gpu_fd = resource_manager_->GetNativeBufferHandler()->GetFd();
```

Comment:
@shuangwan01  Please use ITRACE here since it is an error in HWC. the log here is just for information. The log should be "... skip layer_index: %u for hyper dma buffer sharing"

Reply:
Once these changes are made, this should be good to merge.